libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/api/MapboxRouteLineApi.kt

```diff
@@ -204,6 +205,27 @@ class MapboxRouteLineApi(
     private var lastPointUpdateTimeNano: Long = 0
     private val routeFeatureData: MutableList<RouteFeatureData> = mutableListOf()
     private val jobControl = InternalJobControlFactory.createDefaultScopeJobControl()
+
+    // We had a bug that when styleInactiveRouteLegsIndependently was enabled but we first
```

Comment:
I couldn't reproduce the issue in tests with the new solution (it gets complicates with coroutines in unit tests), so I moved the description here.